oh-my-bash.sh

```diff
@@ -144,6 +144,12 @@ unset -v _omb_init_files _omb_init_file
 # Load the theme
 if [[ $OSH_THEME == random ]]; then
   _omb_util_glob_expand _omb_init_files '"$OSH"/themes/*/*.theme.sh'
+
+  # Remove ignored themes from the list
+  for theme in random ${OMB_THEME_RANDOM_IGNORED[@]}; do
+    _omb_init_files=("${_omb_init_files[@]/$theme}")
```

Comment:
What is contained in _omb_init_files is not a theme name but the theme filename, so the element cannot be removed by `${_omb_init_files/$theme}`.